variantlib/models/variant.py

```diff
@@ -194,11 +193,23 @@ def __post_init__(self) -> None:
         # Execute the validator
         super().__post_init__()
 
+    @property
+    def is_null_variant(self) -> bool:
+        """
+        Check if the variant is a null variant.
+        A null variant is a variant with no properties.
+        """
+        return self.hexdigest == "0" * VARIANT_HASH_LEN
```

Comment:
This seems to be checking a secondary property. I think that by definition, a "null variant" has no properties, so that's what we should check here.

```suggestion
        return not self.properties
```